### ISSUE TITLE
Rehabilitate `IntervalParameters.pi` for use in trig functions

### ISSUE THREAD

**Issue by lbenet** [dpsanders/ValidatedNumerics.jl#62]
`IntervalParameters.pi` is not really used, and whenever it is redefined, it calls `make_interval(BigFloat,pi)`. So I think we can get rid of it quite safely.

@dpsanders Or, have I missed something and we really need it?

**Comment by dpsanders** [comment]
If it's not used, then let's just get rid of it!
Isn't it used in `sin` etc.?

**Comment by dpsanders** [comment]
The idea was that it would be used in `sin` etc. so that the uses of `pi` in `find_quadrants` would use the current precision, not an arbitrary initial fixed precision. I think this needs to be fixed.

**Comment by lbenet** [comment]
This is already achieved by `make_interval`:
```julia
julia> using ValidatedNumerics
julia> ValidatedNumerics.make_interval(Float64, pi)
[3.141592653589793, 3.1415926535897936]
julia> ValidatedNumerics.make_interval(BigFloat, pi)
[3.141592653589793238462643383279502884197169399375105820974944592307816406286198, 3.141592653589793238462643383279502884197169399375105820974944592307816406286233]₂₅₆
julia> set_interval_precision(53)
53
julia> ValidatedNumerics.make_interval(BigFloat, pi)
[3.1415926535897931, 3.1415926535897936]₅₃
```

I think we can safely take pi out of `interval_parameters` (thus solving #66) and define:
```julia
get_pi(::Type{BigFloat}) = make_interval(BigFloat, pi)
get_pi(::Type{Float64}) = float_pi
```
without side effects; `float_pi` is a constant defined as `make_interval(Float64, pi)`. 

We could also define `get_pi(::Type{Float64}) = make_interval(Float64, pi)`, mimicking e.g. `half_pi(::Type{Float64})`.

**Comment by dpsanders** [comment]
The problem with this is that `make_interval(BigFloat, pi)` is liable to be quite slow. This was the reason for wanting to store the result of the calculation. But probably it's a reasonable solution.

**Comment by dpsanders** [comment]
We could also change the function name `get_pi` to `Pi`

**Comment by lbenet** [comment]
> The problem with this is that `make_interval(BigFloat, pi)` is liable to be quite slow. This was the reason for wanting to store the result of the calculation. But probably it's a reasonable solution.

I'm not quite sure; if you are not changing precision too often, once `get_pi` is compiled, it acts as a constant. But you are right for the case of an often change of `BigFloat` precision.

> We could also change the function name `get_pi` to `Pi`

What about `pi_interval` or `set_pi`? In "trigonometric_functions.jl" we use `half_pi` and `two_pi`, and for `atan2` a couple of other intervals is needed, which may be worth defining in this function like manner; we should probably be consistent with the names, and perhaps move the definition of `get_pi` to this file.

**Comment by dpsanders** [comment]
Fixed by #69.
